### ISSUE TITLE
Unable to retrieve light level from motion sensors

### ISSUE THREAD

**Issue by leissnerj** [jbasen/Crestron-Hubitat-Integration#6]
Hi

I'm using a Philips Hue Motion Sensor which reports light level, but im unable to pass that feedback to crestron.
I've tried the different types of sensor modules provided, but none of them retrieves the feedback for light level.

My best guess is that the property is named illuminance in hubitat, and the crestron module is trying to parse something different.

![bild](https://user-images.githubusercontent.com/32220771/115145036-02383d00-a050-11eb-8be9-a738a2d26dc7.png)

**Comment by jbasen** [comment]
I haven't tested with a Hue Motion Sensor.  Can you provide the full json reported by the Maker API?

However, I can tell you that the code fully supports the Aeotec Multisensor 6 which includes motion, light, along with 4 additional sensors.  

Thanks

**Comment by leissnerj** [comment]
Thanks, providing json report here.

[philips hue motion sensor.zip](https://github.com/jbasen/Crestron-Hubitat-Integration/files/6331606/philips.hue.motion.sensor.zip)

**Comment by leissnerj** [comment]
After some fiddling in habitat I've got it working
Looks like the error was on my hub.
I'm now using the Hubitat Mulit-Sensor v3 and it's reporting motion, temperature, illuminance & battery.

I had to change the device type on the sensor in hubitat away from hue sensor and then back to hue sensor.
Really odd, but working quite well. Fast update on motion and illuminance. Great work on these modules!
